magento/magento2
# Fix issue 19499 - Catalog bundle product Bundle Items quantity textbox width

### Description (*)

Catalog bundle product Bundle Items quantity textbox width set
Currently, quantity textbox width is very large it should be small as per other quantity boxes.
Please review link : http://prntscr.com/lp9jpw

### Fixed Issues (if relevant)

1.magento/magento2#19499 : Catalog bundle product Bundle Items quantity textbox width

### Manual testing scenarios (*)

1.Catalog -> Product -> add/edit (Bundle product)
2.Bundle Items -> add options -> adding product
3.Check quatity box


### Contribution checklist (*)
